keps/sig-network/1669-proxy-terminating-endpoints/README.md

```diff
@@ -206,6 +207,11 @@ and internalTrafficPolicy.
 * Metrics to publish how many Services/Endpoints are routing traffic to terminating endpoints.
 * Manual or automated rollback testing (see [Test Plan](#test-plan) section)
 
+#### GA
+
+* Feedback from users in issue [85643](https://github.com/kubernetes/kubernetes/issues/85643) that enabling ProxyTerminatingEndpoints can achieve zero downtime rolling updates when using `externalTrafficPolicy: Local`
+* E2E tests demonstrating that rolling updates can be performed with negligible downtime (ideally 100%, but may not feasible without flakiness) with a Service LoadBalancer when using externalTrafficPolicy: Local
```

Comment:
This test was already added here https://github.com/kubernetes/kubernetes/pull/114052, but there's some follow-up work to increase the minimum target success rate

Reply:
We have seen runs achieve 100% success rate though: https://github.com/kubernetes/kubernetes/pull/114052#issuecomment-1337538200